web/pandas/about/sponsors.md

```diff
@@ -0,0 +1,33 @@
+# Sponsors
+
```

Comment:
There is currently no mention of Tidelift? (here or on the donate or contribute pages)

(the current website has a very brief one: "You can support pandas by becoming a Tidelift subscriber." on the donate page